webapp/components/wpt-results.html

```diff
@@ -258,6 +271,35 @@
         return /(\.(html|htm|py|svg|xhtml|xht|xml)$)/.test(path);
       }
 
+      computeSourcePath(path, manifest) {
+        if (!this.computePathIsATestFile(path) || !manifest) {
+          return path;
+        }
+        const itemSets = TEST_TYPES.map(t => manifest.items[t]).filter(i => i);
+        for (const items of itemSets) {
+          const key = Object.keys(items).find(k => items[k].find(i => i[0] === path));
+          if (key) {
+            // Ensure leading slash.
+            return key.indexOf('/') === 0 ? key : `/${key}`;
```

Comment:
Nit: `key.startsWith('/')` should be a teeny bit faster and looks nice.

Reply:
Done.